engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/stores/ExecutionStore.scala

```diff
@@ -270,22 +272,15 @@ sealed abstract class ExecutionStore private[stores](statusStore: Map[JobKey, Ex
         internalUpdates = internalUpdates ++ key.nonStartableOutputKeys.map(_ -> Unstartable) + (key -> Unstartable)
       }
 
-      key match {
```

Comment:
There's a comment above, on line 262, mentioning `WaitingForQueueSpace`. I think it should be deleted too.